matching-notification-api/src/main/java/org/phenotips/matchingnotification/storage/internal/DefaultMatchStorageManager.java

```diff
@@ -502,6 +503,34 @@ private void copyToHistory(Session session, Collection<PatientMatch> matches)
         }
     }
 
+    /**
+     * Load {@code match} with specified parameters.
+     *
+     * @param referencePatientId id of reference patient
+     * @param referenceServerId id of the server that hosts reference patient
+     * @param matchedPatientId id of matched patient
+     * @param matchedServerId id of the server that hosts matched patient
+     * @return match
+     */
+    private PatientMatch loadMatchBetweenPatients(String referencePatientId, String referenceServerId,
```

Comment:
maybe place it next to `loadMatchesbyId()` in the file (since the methods are related) or next to `updateNotificationHistory()` since that is the only method that uses it? Otherwise it is in a random place in the file